src/sgame/botlib/bot_nav_edit.cpp

```diff
@@ -427,6 +427,132 @@ class AddconCmd: public Cmd::StaticCmd
 	}
 };
 
+static int findClosestNavcon( OffMeshConnections &cons, rVec &targetPoint )
+{
+	int resultIndex = -1;
+	float resultDistance = std::numeric_limits<float>::max();
```

Comment:
Shouldn't there be some distance limit instead of returning one arbitrarily far away?

Reply:
There is a distance limit, he navcon's radius. The function's name is a bit misleading. It only searches for the closest if two or more navcon circles overlap (this happens).